Add MediaStreamTrackEvent.track (#25231)
mdn/content: files/en-us/web/api/mediastreamtrackevent/index.md

_Also inherits properties from its parent interface, {{domxref("Event")}}._

- {{domxref("MediaStreamTrackEvent.track")}} {{ReadOnlyInline}}

>   - : Returns a {{domxref("MediaStreamTrack")}} object representing the track which is concerned by the event.

Reviewer: ```suggestion
  - : Returns a {{domxref("MediaStreamTrack")}} object representing the track associated with the event.
```
